style/properties/longhands/position.mako.rs

```diff
@@ -93,33 +92,19 @@ ${helpers.single_keyword(
     affects="layout",
 )}
 
-% if engine in "servo":
-    // FIXME: Update Servo to support the same Syntax as Gecko.
-    ${helpers.single_keyword(
-        "justify-content",
-        "start flex-start stretch end flex-end center space-between space-around space-evenly",
-        engines="servo",
-        servo_pref="layout.flexbox.enabled",
-        extra_prefixes="webkit",
-        spec="https://drafts.csswg.org/css-align/#propdef-justify-content",
-        animation_value_type="discrete",
-        servo_restyle_damage = "reflow",
-        affects="layout",
-    )}
-% endif
-% if engine == "gecko":
-    ${helpers.predefined_type(
-        "justify-content",
-        "JustifyContent",
-        "specified::JustifyContent(specified::ContentDistribution::normal())",
-        engines="gecko",
-        spec="https://drafts.csswg.org/css-align/#propdef-justify-content",
-        extra_prefixes="webkit",
-        animation_value_type="discrete",
-        servo_restyle_damage="reflow",
-        affects="layout",
-    )}
+${helpers.predefined_type(
+    "justify-content",
+    "JustifyContent",
+    "specified::JustifyContent(specified::ContentDistribution::normal())",
+    engines="gecko servo",
+    spec="https://drafts.csswg.org/css-align/#propdef-justify-content",
+    extra_prefixes="webkit",
+    animation_value_type="discrete",
+    servo_restyle_damage="reflow",
+    affects="layout",
+)}
```

Comment:
It's great to unite these, but we shouldn't lose the `servo_pref` setting here in the process.

Reply:
Sure. Blitz will just enable both, so that works for us. I can go through and update the flags to use the flexbox flag for the case where it is needed for both. Although it wouldn't be that hard to add support for multiple flags if that was desirable (just swap out the `Option<&str>` for `&[&str]` (or `Option<&[&str]>`)